src/commands.cc

```diff
@@ -1381,7 +1382,7 @@ const CommandDesc debug_cmd = {
             auto& keymaps = context.keymaps();
             auto user_modes = keymaps.user_modes();
             write_to_debug_buffer("Mappings:");
-            for (auto& mode : concatenated(modes, user_modes) | gather<Vector<String>>())
+            for (auto& mode : concatenated(modes, user_modes | transform([](auto& i) { return i.key; })) | gather<Vector<String>>())
```

Comment:
I believe you can use `transform(&UserModeList::Item::key)` or just `transform(&UserMode::name)` if you use the struct suggested later.